src/workflows-app/FeaturedWorkflows.ts

```diff
@@ -129,21 +131,30 @@ export const FeaturedWorkflows = ({
             const {
               cbasProxyUrlState: { state },
             } = await loadAppUrls(workspaceId, 'cbasProxyUrlState');
+            const importMethod = (state, method) => {
```

Comment:
This felt odd adding a nested function, but putting it anywhere else I was running into static analysis errors about the `useCallback` closure... would love any feedback on improving this